projects/js-packages/publicize-components/src/components/form/share-count-info.tsx

```diff
@@ -12,40 +12,52 @@ import styles from './styles.module.scss';
 import { useAutoSaveAndRedirect } from './use-auto-save-and-redirect';
 
 export const ShareCountInfo: React.FC = () => {
-	const showShareLimits = useSelect( select => select( socialStore ).showShareLimits(), [] );
+	const { showShareLimits, hasPaidFeatures } = useSelect( select => {
+		const store = select( socialStore );
+		return {
+			showShareLimits: store.showShareLimits(),
+			hasPaidFeatures: store.hasPaidFeatures(),
+		};
+	}, [] );
+
 	const { noticeType, usedCount, scheduledCount, remainingCount } = useShareLimits();
 	const autosaveAndRedirect = useAutoSaveAndRedirect();
 
-	if ( ! showShareLimits ) {
+	if ( ! showShareLimits && hasPaidFeatures ) {
```

Comment:
This will show the nudge on simple sites because the `hasPaidFeatures` check now only looks for advanced plan features that simple sites won't have.

Reply:
Yes, I think an early return for WPCOM sites should be the fix for this.

Reply:
Done in 66ee9b5, thanks!